frontend/src/hooks/useForm.js

```diff
@@ -0,0 +1,13 @@
+import { useState } from 'react'
+
+export default function useForm(initialInput) {
+  const [formData, setFormData] = useState(initialInput)
+
+  return { handleInputChange, formData }
+
+  function handleInputChange(e) {
+    const fieldName = e.target.name
+    const fieldValue = e.target.value
+    setFormData({ ...formData, [fieldName]: fieldValue })
+  }
+}
```

Comment:
Is it necessary to have a custom hook here for your purporse?

Reply:
Maybe not, but now it's there 😂